pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -190,6 +190,34 @@ func toKubeRuntimeStatus(status *runtimeapi.RuntimeStatus) *kubecontainer.Runtim
 	return &kubecontainer.RuntimeStatus{Conditions: conditions}
 }
 
+// toKubeRuntimeConfig converts the runtimeapi.ActiveRuntimeConfig to kubecontainer.RuntimeConfigInfo
+func toKubeRuntimeConfig(config *runtimeapi.ActiveRuntimeConfig) *kubecontainer.RuntimeConfigInfo {
+	usernsConfig := config.GetUserNamespaceConfig()
+	if usernsConfig == nil {
+		return nil
+	}
+	uidMappings := usernsConfig.GetUidMappings()
+	if uidMappings == nil || len(uidMappings) == 0 {
+		return nil
+	}
+	gidMappings := usernsConfig.GetGidMappings()
+	if gidMappings == nil || len(gidMappings) == 0 {
+		return nil
+	}
+
+	userNSConfig := kubecontainer.UserNamespaceConfigInfo{
+		UidMapping: kubecontainer.UserNSMapping{
+			ContainerID: uidMappings[0].ContainerId,
```

Comment:
it still seems incorrect to drop all items other than the first here. isn't this losing mapping range info provided by the runtime?